app/components/channel_drawer/channel_drawer.js

```diff
@@ -395,47 +370,24 @@ export default class ChannelDrawer extends Component {
         const androidTop = isLandscape ? ANDROID_TOP_LANDSCAPE : ANDROID_TOP_PORTRAIT;
         const iosTop = isLandscape ? IOS_TOP_LANDSCAPE : IOS_TOP_PORTRAIT;
 
+        // Use a polyfill on iOS and native DrawerLayoutAndroid on Android
+    
+        const Drawer = Platform.OS === 'android' ? DrawerLayoutAndroid : DrawerLayout;
+        const additionalProps = Platform.OS == 'android' ? {} : {useNativeAnimations: true};
+
         return (
             <Drawer
                 ref='drawer'
-                onOpenStart={this.handleDrawerOpenStart}
-                onOpen={this.handleDrawerOpen}
-                onClose={this.handleDrawerClose}
-                onCloseStart={this.handleDrawerCloseStart}
-                captureGestures='open'
-                type='static'
-                acceptTap={true}
-                acceptPanOnDrawer={false}
-                disabled={false}
-                content={this.renderContent()}
-                tapToClose={true}
-                openDrawerOffset={openDrawerOffset}
-                onRequestClose={this.closeChannelDrawer}
-                panOpenMask={0.2}
-                panCloseMask={openDrawerOffset}
-                panThreshold={0.25}
-                acceptPan={true}
-                negotiatePan={true}
-                useInteractionManager={false}
-                tweenDuration={100}
-                tweenHandler={this.handleDrawerTween}
-                elevation={-5}
-                bottomPanOffset={Platform.OS === 'ios' ? ANDROID_TOP_LANDSCAPE : IOS_TOP_PORTRAIT}
-                topPanOffset={Platform.OS === 'ios' ? iosTop : androidTop}
-                styles={{
-                    main: {
-                        shadowColor: '#000000',
-                        shadowOpacity: 0.4,
-                        shadowRadius: 12,
-                        shadowOffset: {
-                            width: -4,
-                            height: 0,
-                        },
-                    },
-                }}
+                drawerWidth={Platform.OS === 'android' ? 300 : 340}
```

Comment:
why the different width between the platforms?

Reply:
on iOS it cuts the searchbox when set to 300, check it yourself. 340 is a magic number that makes it render correctly on iPhones as I tested.